### ISSUE TITLE
Добавить решение к каждому упражнению.

### ISSUE THREAD

**Issue by MAGLeb** [Hexlet/hexlet-sicp#1321]
http://community.schemewiki.org/?sicp-solutions

Возможность смотреть лучшее решение. На сайте выше есть очень хорошие решения для каждого упражнения.

**Comment by fey** [comment]
по сути это https://github.com/Hexlet/hexlet-sicp/issues/1312
т.е. нужно добавить вкладку с решением.

**Comment by MAGLeb** [comment]
Наверное да, в #1312 не совсем понятно что будет в этой вкладке.

**Comment by fey** [comment]
У нас есть "эталонное" решение - по сути то, что выше по ссылке. (если вы учились на Хекслете или на Code Basics, то это похоже).
Вот это решение нужно научиться выводить (и отдавать с бекенда)

**Comment by MAGLeb** [comment]
Правильно понимаю, у вас есть "эталонное" решение, где-то на сервере, и нужно давать доступ к его просмотру тем, кто правильно выполнил задание?

**Comment by fey** [comment]
Да, все верно, здесь две задачи, первая это сделать вкладку с тестами если студент выполнил упражнение, либо по таймеру
И второе - отдавать решение с бекенда

**Comment by MAGLeb** [comment]
Вы не будете против если я попробую заняться этой задачей?

**Comment by fey** [comment]
Не против :)

**Comment by MAGLeb** [comment]
Есть несколько вопросов.
1. Не могу запушить ветку, чтобы создать PR и вести беседу там, нет доступов. Может быть как-то по другому нужно создать PR?
2. Когда собираю проект локально выполняя следующие команды
```
make compose-setup
make compose-start
```
фронтенд другой.
![image](https://user-images.githubusercontent.com/52084975/183085125-3f18b123-a851-4310-86bd-a0275d2bb101.png)

**Comment by fey** [comment]
Вижу, что разобрались. Все верно, нужно пушить в свой форк.
Вот видео хорошее на тему https://www.youtube.com/watch?v=pUT3mx1ZRUM

**Comment by acidmaksim** [comment]
@MAGLeb Как дела с задачей?

**Comment by MAGLeb** [comment]
https://github.com/Hexlet/hexlet-sicp/pull/1333

ПР смержен. Забыл закрыть таску.
